src/bin/__tests__/check-engine.js

```diff
@@ -0,0 +1,9 @@
+import test from "ava"
+import checkEngine from "../check-engine"
+
+test("should not throw when sastifies", (t) => {
+  t.notThrows(checkEngine)
+})
+
+// process.version is READONLY
```

Comment:
make an util to read process.version that allow you, while TESTING, to use a process.env.FAKE_NODE_VALUE ?!